deployment/stack.py

```diff
@@ -439,6 +437,8 @@ def make_template(config, config_yaml):
             'SecurityGroups': [
                 functions.get_att('ELBSecurityGroup', 'GroupId')],
             "Tags": instance_tags,  # todo: Should be different?
+            'AvailabilityZones': [
+                functions.get_att('WebInstance', 'AvailabilityZone')],
```

Comment:
A nitpick: if an open bracket is on a line by itself then the closing one should be as well